examples/stable_diffusion_v2/ldm/modules/diffusionmodules/openaimodel.py

```diff
@@ -154,14 +154,14 @@ def __init__(
             self.h_upd = self.x_upd = self.identity
 
         self.emb_layers = nn.SequentialCell(
-            nn.SiLU().to_float(self.dtype),
+            nn.SiLU().to_float(ms.float32),
```

Comment:
can influence the performance of other SD models. Need to check whether using fp16 is also fine.